plaidml/exec/exec.h

```diff
@@ -135,6 +162,11 @@ class Binder {
   using BindingMap = std::map<edsl::TensorRef, Buffer>;
 
  public:
+  ///
+  /// Constructs a Binder.
+  /// By default, this constructor uses the environment variables `PLAIDML_DEVICE` and `PLAIDML_TARGET` to specify your
```

Comment:
This constructor doesn't really depend on env vars. It's more like it depends on those settings to have been configured. The default values for those settings default to the env var values.